docs/deploy-module-from-source-code.md

```diff
@@ -0,0 +1,76 @@
+# Deploy Module from source code
+
+[Deploy Platform from source code](deploy-from-source-code.md)
+
+## Installation
+
+To get started locally, follow these instructions:
+1. Make fork from the latest module source code from https://github.com/VirtoCommerce/{module-name}.git
+1. Clone to your local computer using `git`
+1. Make sure that you have All Prerequisites installed
+
+## Building Module
+### Build Backend
+To make a local build:
+1. Open console
+```console
+cd src/VirtoCommerce.{module-name}.Web
+```
+2. Build 
+```console
+dotnet build -c Development
+```
+
+Or use Visual Studio:
+1. Open VirtoCommerce.{module-name}.sln in Visual Studio 
+2. Build Solution
+
```

Comment:
Add 
**Note** Recommend to install [WebPack Task Runner](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.WebPackTaskRunner) and run webpack tasks from Visual Studio.